codex-rs/.gitignore

```diff
@@ -1,4 +1,5 @@
 /target/
+/target-*/
```

Comment:
Not technically related to this PR but I added `"rust-analyzer.cargo.targetDir": "target-ide"`  to my vs code settings.json so terminal builds won't block on IDE updates after edits.